content/zh-cn/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -248,7 +247,7 @@ kubectl apply -f cassandra-statefulset.yaml
    -->
    响应应该与此类似：
 
-   ```
+   ```shell
```

Comment:
shell -> none

Reply:
I was very puzzled about this adjustment at first, but I chose to synchronize with English, because English is marked like this

Reply:
In Mandarin we fix formatting like this and backport to English. Reverting the fix is not a good idea.